src/librbd/deep_copy/ObjectCopyRequest.cc

```diff
@@ -249,7 +249,7 @@ void ObjectCopyRequest<I>::send_update_object_map() {
   auto dst_image_ctx = m_dst_image_ctx;
   bool sent = dst_image_ctx->object_map->template aio_update<
     Context, &Context::complete>(dst_snap_id, m_dst_object_number, object_state,
-                                 {}, {}, false, ctx);
+                                 {}, {}, false, true, ctx);
```

Comment:
If I am not missing something I think if we blindly set force for every update we will end up with many unnecessary updates. I suppose we need to set the force flag only when the (new) object_state is NONEXISTENT and the current state in the map is not NONEXISTENT (`object_may_exist` returned `true`)

Reply:
We need to set the force flag whenever the new state is NONEXISTENT because this code path does not set the state to PENDING.

Reply:
Ok. Does it contradict what I wrote above? Right now you unconditionally set the force flag (for all updates) and I propose to set it only for this particular case.

Reply:
Done.